src/ol/README.md

```diff
@@ -34,11 +34,10 @@ new Map({
 
 See the following examples for more detail on bundling OpenLayers with your application:
 
- * Using [Rollup & Uglify](https://gist.github.com/tschaub/8beb328ea72b36446fc2198d008287de)
+ * Using [Rollup](https://github.com/openlayers/ol-rollup)
  * Using [Rollup & Closure Compiler](https://gist.github.com/tschaub/32a5692bedac5254da24fa3b12072f35)
```

Comment:
Shouldn't be also removed?

Reply:
Good catch, I meant to leave in the link to @ahocevar's example:
```
* Using [Webpack & Closure Compiler](https://gist.github.com/ahocevar/8ceafc6293455ba491dd9be12c15761f)
```

Reply:
Instead, I'll remove all the Closure Compiler examples.

Reply:
Thanks :-)